tests/test_modeling_tf_lxmert.py

```diff
@@ -751,20 +875,68 @@ def test_saved_model_creation_extended(self):
+                )
+                self.assertEqual(
+                    len(vision_attentions),
+                    self.model_tester.num_hidden_layers["vision"],
+                )
+                self.assertEqual(
+                    len(cross_encoder_attentions),
+                    self.model_tester.num_hidden_layers["cross_encoder"],
+                )
 
-                attentions = [language_attentions, vision_attentions, cross_encoder_attentions]
+                attentions = [
+                    language_attentions,
+                    vision_attentions,
+                    cross_encoder_attentions,
+                ]
                 attention_shapes = [
-                    [self.model_tester.num_attention_heads, encoder_seq_length, encoder_key_length],
+                    [
+                        self.model_tester.num_attention_heads,
+                        encoder_seq_length,
+                        encoder_key_length,
+                    ],
                     [
                         self.model_tester.num_attention_heads,
                         self.model_tester.num_visual_features,
                         self.model_tester.num_visual_features,
                     ],
-                    [self.model_tester.num_attention_heads, encoder_key_length, self.model_tester.num_visual_features],
+                    [
+                        self.model_tester.num_attention_heads,
+                        encoder_key_length,
+                        self.model_tester.num_visual_features,
+                    ],
                 ]
 
                 for attention, attention_shape in zip(attentions, attention_shapes):
                     self.assertListEqual(list(attention[0].shape[-3:]), attention_shape)
+
+
+@require_tf
+class TFLxmertModelIntegrationTest(unittest.TestCase):
+    @slow
+    def test_inference_masked_lm(self):
+        model = TFLxmertModel.from_pretrained("unc-nlp/lxmert-base-uncased")
+        input_ids = tf.constant([[101, 345, 232, 328, 740, 140, 1695, 69, 6078, 1588, 102]])
+
+        num_visual_features = 10
+        _, visual_feats = np.random.seed(0), np.random.rand(1, num_visual_features, model.config.visual_feat_dim)
+        _, visual_pos = np.random.seed(0), np.random.rand(1, num_visual_features, 4)
+        visual_feats = tf.convert_to_tensor(visual_feats, dtype=tf.float32)
+        visual_pos = tf.convert_to_tensor(visual_pos, dtype=tf.float32)
+        output = model(input_ids, visual_feats=visual_feats, visual_pos=visual_pos)[0]
+        expected_shape = [1, 11, 768]
+        self.assertEqual(expected_shape, output.shape)
+        expected_slice = tf.constant(
+            [
+                [
+                    [0.24170142, -0.98075, 0.14797261],
+                    [1.2540525, -0.83198136, 0.5112344],
+                    [1.4070463, -1.1051831, 0.6990401],
+                ]
+            ]
+        )
+        tf.debugging.asset_near(output[:, :3, :3], expected_slice, atol=1e-4)
```

Comment:
This should be `assert_near`
```suggestion
        tf.debugging.assert_near(output[:, :3, :3], expected_slice, atol=1e-4)
```